docs/getting_started/get_started.md

```diff
@@ -43,3 +76,49 @@ You can access your user settings on the right side of the top bar:
 
 Here you can change your user password and define your user preferences,
 such as preferred timezone, date and time format, or Back Office language.
+
+**Location**
+
+|Setting|Description|
+--------|-----------|
+|Default currency|Sets the default currency used in the Back Office.|
+|Toggle In-Context translation feature|Enables or disables integration with Crowdin to navigate the interface while translating.|
+|User Time Zone|Sets time zone in the Back Office.|
+|Date and time format|Sets format used in the Back Office.|
+|Language|Sets the default language used in the Back Office.|
+
+
+**Content authoring**
+
+|Setting|Description|
+--------|-----------|
+|[Autosave draft every given period](../content_management/content_versions.md/#autosave)|Enables or disables autosaving drafts.|
+|Seconds till next draft autosave|Sets time period for next autosave.|
+|Enable character count in Online Editor|Enables or disables charactes count.|
+|Automatically open block settings in builders|Enables or disables the behavior of blocks used in builders.|
+
+**Browsing**
+
+|Setting|Description|
+--------|-----------|
+|Number of items displayed in the table|Sets the number of items displayed in sub-items.|
```

Comment:
```suggestion
|Number of items displayed in the table|Sets the number of items displayed in the sub-items list.|
```

Reply:
Not only list, it also refers to grid view items.